webapp/src/ts/main.ts

```diff
@@ -1,3 +1,14 @@
+import { IntegrationApiService } from '@mm-services/integration-api.service';
+
+
+const logger = reducer => {
```

Comment:
This was originally over in the `app.module.ts`. Pulled it over here where it is being used now.